types/lodash/scripts/package.json

```diff
@@ -3,8 +3,7 @@
   "name": "lodash-scripts",
   "version": "0.0.1",
   "scripts": {
-    "generate": "ts-node generate-fp",
-    "fp": "ts-node generate-fp"
+    "generate": "ts-node generate-modules && ts-node generate-fp"
```

Comment:
At one point this was part of the generate script, but I think it was removed for some reason (although I can't remember why). Make sure you test this script and make sure it works properly (if you haven't already).

Reply:
Yes, it worked and the changes such as
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/42220/files#diff-3b06535aaa6734aef5f334131f1787ea
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/42220/files#diff-ab5db76f7fdb8d87752d708ccc71c8ed
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/42220/files#diff-c10914abbc618cd230274d7dcde19eca
are the result of `generate-modules.ts`.

I will try to find out the reason from logs somewhere, but if it was intended to be removed, what's the correct way to generate and keep up-to-date `lodash.*` modules?